### DISCUSSION TITLE
Change UTXOs orange arrow seems confusing

### DISCUSSION THREAD

**Discussion by BTCBellyButton** [cryptoadvance/specter-desktop#761] [Specter UI/UX]
This wallet has only 1 UTXO and after scanning the UTXOs it shows the orange arrow:

![image](https://user-images.githubusercontent.com/68573637/101505331-db4b5b00-3939-11eb-8dc3-28199711db9a.png)

It's confusing to me. It looks like a spent transaction...

**Comment by ben-kaufman** [comment]
That's because it is a change coming from a spend transaction you have made.
So since it's part of a send transaction (just the change left from it) it shows as send. Does that make sense?

**Comment by BTCBellyButton** [comment]
I see your point but it is confusing IMO from a UX perspective.
Change is something that comes back into your wallet. It doesn't go out.

I'd pick a different color for the change, if you want to differentiate those UTXOs, but I believe the arrow should be pointing down like the incoming transactions.

**Comment by k9ert** [comment]
I would agree to @BTCBellyButton . This might be a bigger thing to change, though.
 @stepansnigirev @moritzwietersheim  what do you think?

**Comment by ben-kaufman** [comment]
Yeah I see that makes sense, I'd better assign it a new icon though than the receive one though...
